diktat-rules/src/test/kotlin/org/cqfn/diktat/util/LintTestBase.kt

```diff
@@ -22,26 +22,48 @@ open class LintTestBase(private val ruleSupplier: (rulesConfigList: List<RulesCo
      * @param lintErrors expected errors
      * @param rulesConfigList optional override for `this.rulesConfigList`
      * @param fileName optional override for file name
+     * @see lintResult
      */
-    @OptIn(FeatureInAlphaState::class)
     fun lintMethod(@Language("kotlin") code: String,
                    vararg lintErrors: LintError,
                    rulesConfigList: List<RulesConfig>? = null,
                    fileName: String? = null
     ) {
+        lintResult(code, rulesConfigList, fileName)
+            .assertEquals(*lintErrors)
+    }
+
+    /**
+     * Lints the [code] and returns the errors collected, but (unlike
+     * [lintMethod]) doesn't make any assertions.
+     *
+     * @param code the code to check.
+     * @param rulesConfigList an optional override for `this.rulesConfigList`.
+     * @param fileName an optional override for the file name.
+     * @return the list of lint errors.
+     * @see lintMethod
+     */
+    @OptIn(FeatureInAlphaState::class)
+    protected fun lintResult(
+        @Language("kotlin") code: String,
```

Comment:
What does this annotation do? Will IDEA highlight strings used as this function parameter as if it was Kotlin code?

Reply:
Yes, exactly.